Storage/Rust/RustAutofill.swift

```diff
@@ -61,6 +61,27 @@ public struct UnencryptedCreditCardFields {
                                          ccExpYear: self.ccExpYear,
                                          ccType: self.ccType)
     }
+
+    public func toFakeCreditCard() -> CreditCard {
+        let rustKeys = RustAutofillEncryptionKeys()
+        let ccNumberEnc = rustKeys.encryptCreditCardNum(creditCardNum: self.ccNumber)
+        let fakeCreditCard = CreditCard(guid: "",
```

Comment:
Same here, lets not call it fake credit card and instead use words like `plainText` or `convertedCreditCard`